### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by butalas**
I finished the login page by implementing the designs from Figma. I also attached a screenshot!

img_6053

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,11 +1,38 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View} from 'react-native';
+import { Font } from 'expo';
+
 
 export default class App extends React.Component {
```
Comment (ijemmao) [inline App.js]: Make sure to move this code into its own file. Probably wanna call it `login.js`

File: App.js [hunk 2]
```diff
@@ -17,5 +44,118 @@ const styles = StyleSheet.create({
...
+    height: 370,
+    left: 14,
+    top: 250,
+    borderRadius: 25,
+    shadowOffset: {width: 0, height: 5},
+    shadowColor: '#000000',
+    shadowOpacity: 0.25,
+    shadowRadius: 5,
+  },
+  subText1: {
```
Comment (ijemmao) [inline App.js]: `subText1` isn't descriptive enough - try to have something like `loginBox` or `loginText`. Also, don't include numbers in the name of the styles.

File: App.js [hunk 3]
```diff
@@ -17,5 +44,118 @@ const styles = StyleSheet.create({
...
+    fontWeight: '200',
+    color: '#605DF1',
+    position: 'absolute',
+    width: 310,
+    height: 101,
+    left: 32,
+    top: 267,
+    textAlign: 'center'
   },
+  typingLine1: {
```
Comment (ijemmao) [inline App.js]: Change the name of this style

File: App.js [hunk 4]
```diff
@@ -17,5 +44,118 @@ const styles = StyleSheet.create({
...
+    flexDirection: 'row',
+    position: 'absolute',
+    backgroundColor: '#6DC4E0',
+    width: 274,
+    height: 2,
+    left: 50,
+    top: 361
+  },
+  subText2: {
```
Comment (ijemmao) [inline App.js]: Change the name of this style

File: App.js [hunk 5]
```diff
@@ -17,5 +44,118 @@ const styles = StyleSheet.create({
...
+    fontWeight: '200',
+    color: '#605DF1',
+    position: 'absolute',
+    width: 310,
+    height: 101,
+    left: 32,
+    top: 428,
+    textAlign: 'center'
+  },
+  typingLine2: {
```
Comment (ijemmao) [inline App.js]: Change the name of this style

File: screens/login_screen.js [hunk 6]
```diff
@@ -0,0 +1,161 @@
+import React from 'react';
```
Comment (ijemmao) [inline screens/login_screen.js]: Change the file name to just `login.js`.

File: App.js (hunk 1 above)
Comment (butalas) [reply App.js]: Do you mean that I should only have code in `login.js`?

**Comment by butalas** [comment]
I addressed all of your comments and pushed the commit to my branch.

File: App.js (hunk 1 above)
Comment (ijemmao) [reply App.js]: Yup

**Comment by butalas** [comment]
I just changed the `App.js` file to match what's in that file on the master branch.

**Pull request merged**